docs/pages/management/admin/uninstall-teleport.mdx

```diff
@@ -187,6 +186,8 @@ Follow the instructions for your Linux distribution:
   ```
 (!docs/pages/includes/uninstall-teleport-connect-windows.mdx!)
 
+(!docs/pages/includes/uninstall-windows-auth.mdx!)
```

Comment:
@ptgott any idea why this partial is not showing up in the preview?

Reply:
It seems to work on my end!
<img width="963" alt="image" src="https://github.com/user-attachments/assets/cea96d4a-6937-4e6c-8ce1-4a6c6009d165">

Reply:
Odd. I'm certain I checked this page and it did not show up.